easybuild/easyconfigs/k/Kraken/Kraken-1.1.1-foss-2019a-Perl-5.28.1.eb

```diff
@@ -0,0 +1,52 @@
+# Updated from previous config
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PackedBinary'
+
+name = 'Kraken'
+version = '1.1.1'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://ccb.jhu.edu/software/%(namelower)s/'
+description = """Kraken is a system for assigning taxonomic labels to short DNA sequences,
+ usually obtained through metagenomic studies. Previous attempts by other
+ bioinformatics software to accomplish this task have often used sequence
+ alignment or machine learning techniques that were quite slow, leading to
+ the development of less sensitive but much faster abundance estimation
+ programs. Kraken aims to achieve high sensitivity and high speed by
+ utilizing exact alignments of k-mers and a novel classification algorithm."""
+
+# Part is compiled with CXX, the rest is in Perl
+toolchain = {'name': 'foss', 'version': '2019a'}
```

Comment:
I wonder; does this use BLAS? If not, it would be moved to gompic. If it also doesn't use MPI (and I suspect it doesn't), then it could be moved to GCC level.